src/clr-addons/flow-bar/flow-bar.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+export interface ClrFlowBarStep {
+  title: string;
+  enabled: boolean;
+}
+
+@Component({
+  selector: 'clr-flow-bar',
+  templateUrl: './flow-bar.html',
+  host: {
+    '[class.flow-bar]': 'true',
+  },
+})
+export class ClrFlowBar implements OnInit {
+  _activeStep: ClrFlowBarStep;
+
+  @Input('clrSteps') _steps: ClrFlowBarStep[] = [];
+  @Output('clrActiveStepChange') _activeStepChange: EventEmitter<ClrFlowBarStep> = new EventEmitter(false);
+
+  @Input('clrActiveStep')
+  get activeStep() {
+    return this._activeStep;
+  }
+
+  set activeStep(step: ClrFlowBarStep) {
+    this._activeStep = step;
+    this._activeStepChange.emit(this._activeStep);
+  }
+
+  ngOnInit() {
+    // If no active step is set as input or the active step is not enabled, select the first enabled step
+    if (!this._activeStep || !this._activeStep.enabled) {
+      this._activeStep = this._steps.find(step => {
```

Comment:
the initial step should not emit the "activeStepChange" event? just asking to be sure